couchbase/README.md

```diff
@@ -46,25 +46,27 @@ Metadata associated with SignalFx's Couchbase integration can be found <a target
 
 ### USAGE
 
-Below are screen captures of dashboards created for this plugin by SignalFx, illustrating the metrics emitted by this plugin.
+Screen captures below of dashboards created for this plugin by SignalFx illustrate the metrics emitted by this plugin.
 
 For general reference on how to monitor Couchbase, see <a target="_blank" href="http://blog.couchbase.com/monitoring-couchbase-cluster">Couchbase Monitoring</a> and <a target="_blank" href="http://developer.couchbase.com/documentation/server/4.0/monitoring/monitoring-rest.html">Monitor using the REST API</a>.
 
 **Note on bucket metrics**
 
-This plugin emits some metrics about the bucket's performance across the cluster, and some metrics about the bucket's performance per node.
+This plugin emits metrics about bucket performance across the cluster, and also metrics about bucket performance per node.
```

Comment:
is emit the best word we can use here? maybe export, send, generate, create? if this is used across multiple tiles, better to keep consistent, though.

also, i think "plugin" should be "plug-in"

Reply:
Splunk usage dictionary lists "plugin" with no hyphen.